.docker/images/govcms7/settings/settings.php

```diff
@@ -199,3 +199,8 @@ public function __toString() {
   }
 
 }
+
+// X-Frame-Options must mirror environment variable if provided.
+if (getenv('X_FRAME_OPTIONS')) {
+  $conf['x_frame_options'] = getenv('X_FRAME_OPTIONS');
```

Comment:
Slight variation to D8.

In D7: If Seckit x-frame-options is "disabled" then it will fall back to the core value of `SAMEORIGIN`. This differs from D8, which removes the header entirely.

This ensures that the override takes precedence.